File: examples/simpleClass.py
# 単純なクラス例 - Phase E テスト用
class Counter:
    def __init__(self, start: int):
        self.__count: int = start
    
    def increment(self) -> int:
        self.__count = self.__count + 1
        return self.__count

if __name__ == "__main__":
    c: Counter = Counter(0)
    print(c.increment())

File: simpleClass.rs
#[derive(Clone)]
struct Counter {
    count: i64,
}
impl Counter {
    fn increment(&mut self, ) -> i64 {
        self.count = (self.count + 1i64);
        return self.count;
    }
}

fn main() -> () {
    let mut c: Counter = Counter { count: 0i64 };
    println!("{:?}", &c.increment());
}